.github/workflows/lint.yaml

```diff
@@ -46,31 +46,31 @@ jobs:
       - name: Setup NodeJS
         uses: actions/setup-node@v4
         with:
-          node-version: '18'
+          node-version: "18"
       - name: Install TypeSpec compiler
         run: |
           pushd typespec
           npm ci
           popd
       - name: Install autorest
-        run: npm install -g autorest@3.6.1
```

Comment:
Are there ways we could get dependabot to manage this? I know dependabot understands things like `go.mod`, can we teach it to understand versions in other kinds of files?

It would be nice if we could make some improvements that ensure we have **no** hardcoded versions (everything managed by dependabot).

Reply:
I can check this but, as far as I know, **dependabot** doesn't have something like you mentioned above. I will check again, and, if there is, I will create a PR to add that.

Reply:
One idea would be to move this to a `project.json` so Dependabot can manage it for us. Then from the pipeline we'd call `npm install` or `npx`

Reply:
No need to do that in this PR.